src/main/java/org/folio/consortia/config/JpaConfig.java

```diff
@@ -0,0 +1,9 @@
+package org.folio.consortia.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+
+@Configuration
+@EnableJpaAuditing(modifyOnCreate = false)
```

Comment:
I propose to move this annotation to FolioAuditorAware and delete this class.
Also please add class-level java doc of this FolioAuditorAware and specify that getCurrentAuditor() needed to poulate current userId into metadata table

Reply:
fixed